$MirOS: ports/emulators/qemu/patches/patch-block-cow_c,v 1.1 2005/05/03 19:30:03 tg Exp $
--- block-cow.c.orig	Sun Sep  4 17:11:09 2005
+++ block-cow.c	Tue Nov 15 11:37:42 2005
@@ -260,5 +260,7 @@ BlockDriver bdrv_cow = {
     cow_close,
     cow_create,
     cow_is_allocated,
+    NULL,
+    NULL
 };
 #endif
